common/src/cams/session.ts

```diff
@@ -0,0 +1,12 @@
+export const MOCK_AUTHORIZATION_BEARER_TOKEN = 'MOCK_ACCESS_TOKEN_JWT_HEADER.PAYLOAD.SIGNATURE';
```

Comment:
See if we can remove this from the entire codebase now that we have a mock JWT function in MockData.